source/templates.tex

```diff
@@ -6462,7 +6462,8 @@
 
 \pnum
 If a template, a member template or a member of a class template is explicitly
-specialized then that specialization shall be declared before the first use of
+specialized, a declaration of that specialization shall be reachable from
+every use of
 that specialization that would cause an implicit instantiation to take place,
 in every translation unit in which such a use occurs;
 no diagnostic is required.
```

Comment:
```suggestion
that specialization that would cause an implicit instantiation to take place;
no diagnostic is required.
```

Reply:
@opensdh, any thoughts on this suggestion?

Reply:
This is part of [his suggestion](https://github.com/cplusplus/draft/pull/4489#discussion_r572329651).

Reply:
I see, thanks!